src/wallet/wallet.cpp

```diff
@@ -2214,6 +2214,7 @@ SpendableInputs CWallet::FindSpendableInputs(
     KeyIO keyIO(Params());
 
     bool selectTransparent{selector.SelectsTransparent()};
+    bool selectTransparentCoinbase{selector.SelectsTransparentCoinbase()};
```

Comment:
```suggestion
    bool selectTransparentForCoinbase{selector.SelectsTransparentForCoinbase()};
```

Reply:
I don’t understand the addition of `For` in these suggestions. It’s selecting transparent coinbase notes(?) as opposed to transparent utxos or shielded coinbase.

Reply:
Ah, I think I read these functions as non-disjoint. Something can return true for both `SelectsOrchard` and `SelectsSapling`. So, with that, I would say that `SelectsTransparentCoinbase` is correct, and (if anything, but not really) `SelectsTransparent` could be renamed `SelectsTransparentNonCoinbase`.